src/components/Problem/Problem.tsx

```diff
@@ -51,6 +52,25 @@ export const Problem: FC<ProblemProps> = ({ problem }) => {
         destructive: true,
     });
 
+    const problemArchiveConfirmation = useConfirmation({
+        message: problem.archived ? tr('Remove problem from archive?') : tr('Archive problem?'),
+        onAgree: () =>
+            problemUpdateMutation
+                .mutateAsync({
+                    problemId: problem.id,
+                    archived: !problem.archived,
```

Comment:
You can pass only one field - `archived`, everything else stays the same

Reply:
yes but then history items are created for changing all fields